### ISSUE TITLE
Several specs are failing vue to MSSQL incompatibilities

### ISSUE THREAD

**Issue by yivo** [openware/peatio#206]
When Peatio is used with MSSQL some specs are failing:

```
Tests Failures related:

  1) withdraw admin reject withdraw
     Failure/Error: let!(:identity) { create :identity, email: Member.admins.first }
     
     ActiveRecord::RecordInvalid:
       Validation failed: Email has already been taken
     # ./spec/features/admin/withdraw_spec.rb:5:in `block (2 levels) in <top (required)>'
     # ./spec/features/admin/withdraw_spec.rb:3:in `block (2 levels) in <top (required)>'


  5) AccountVersion#optimistically_lock_account_and_save! should raise StaleObjectError if associated account is stale
     Failure/Error:
       expect do
         # `unlock_and_sub_funds('5.0'.to_d, locked: '8.0'.to_d, fee: ZERO)`
         ActiveRecord::Base.connection.execute "update accounts set balance = balance + 3, locked = locked - 8 where id = #{account.id}"
         AccountVersion.optimistically_lock_account_and_create!('13.0'.to_d, '2.0'.to_d, attrs)
       end.to raise_error(ActiveRecord::StaleObjectError)
     
       expected ActiveRecord::StaleObjectError but nothing was raised
     # ./spec/models/account_version_spec.rb:41:in `block (3 levels) in <top (required)>'

  6) Account concurrent lock_funds should raise error on the second lock_funds
     Failure/Error:
       expect do
         ActiveRecord::Base.transaction do
           account1.lock_funds 8, reason: Account::ORDER_SUBMIT
         end
         ActiveRecord::Base.transaction do
           account2.lock_funds 8, reason: Account::ORDER_SUBMIT
         end
       end.to raise_error(ActiveRecord::RecordInvalid)
     
       expected ActiveRecord::RecordInvalid but nothing was raised
     # ./spec/models/account_spec.rb:273:in `block (3 levels) in <top (required)>'

  7) Account after callback should retry the whole transaction on stale object error
     Failure/Error:
       expect do
         expect do
           ActiveRecord::Base.transaction do
             create(:order_ask) # any other statements should be executed
             subject.unlock_and_sub_funds('1.0'.to_d, locked: '2.0'.to_d)
           end
         end.to change(OrderAsk, :count).by(1)
       end.to change(AccountVersion, :count).by(1)
     
       expected #count to have changed by 1, but was changed by 0
     # ./spec/models/account_spec.rb:250:in `block (3 levels) in <top (required)>'
```

**Comment by yivo** [comment]
Please base your branch on master and target your PR to branch devel. Push PR as you start working.

**Comment by yivo** [comment]
Closed via #567.
